### DISCUSSION TITLE
Urgent Assistance Needed: Hacked and Suspended GitHub Account

### DISCUSSION THREAD

**Discussion by 11Cream** [community/community#108020] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Community,

I hope this message finds you well. My name is Darshan(Darsh) Mistry, and I am reaching out with a pressing issue regarding my GitHub account.

On February 2, 2024, at 5:56 AM IST, my GitHub account, under the username 1111darsh, was compromised and subsequently suspended. The primary email associated with this account is 1111darsh@gmail.com.

I received several emails prompting me to reset my password, presumably initiated by the hacker. However, upon attempting to reset my password, I encountered difficulties. Each time I tried to reset it, I received a notification indicating that my account had been suspended.

Simultaneously, unauthorized changes were made to my repository named "cream" under the username 1111darsh. These alterations were executed without my knowledge or consent. I became aware of these activities through email alerts.

I have attached screenshots of the suspicious activities for your reference.

The suspension of my GitHub account is severely hindering my workflow and impeding my ability to collaborate effectively.

I urgently seek assistance from the GitHub community to recover my account and rectify the unauthorized actions taken within my repository. Any guidance or support provided would be greatly appreciated.

Thank you for your attention to this matter.

Ticket ID: 2571360

![github](https://github.com/community/community/assets/27777800/dff72733-8c0d-4bdf-be01-c1c26191e60d)
![mail1](https://github.com/community/community/assets/27777800/d6749159-cb81-4487-a0c5-8dc1613503d8)
![mail2](https://github.com/community/community/assets/27777800/7d1ca5c4-f522-4f12-ae16-69922dafe858)
![mail3](https://github.com/community/community/assets/27777800/e026632a-40f9-469c-b1c1-8b3d4ace9969)
![mail4](https://github.com/community/community/assets/27777800/65194513-5720-4692-a46c-c19e55939ac5)

Sincerely,
Darshan Mistry(Darsh Mistry) 
1111darsh@gmail.com

**Comment by Meetcpatel** [comment]
+1

**Comment by queenofcorgis** [comment, marked as answer]
Hi @11Cream,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
